src/index.html

```diff
@@ -11,6 +11,53 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="/" 
+      method="get"
+      class="block"
+      data-qa="big"
+    >
+      <div class="container">
+        <img
+          src="./images/Search.svg"
+          alt="loop-icon"
+          class="loop"
+        >
+        <label for="bigger">
+          <input
+            type="text"
+            class="big-input input"
```

Comment:
```suggestion
            class="input"
```
good, but you have only 2 inputs
so you can add all styles for `big` input to `.input` class, and then change what you need in `.small-input` for small input